src/components/Stats.astro

```diff
@@ -0,0 +1,336 @@
+      const isNowMobile = window.innerWidth <= 768;
+
+      // Remove all active classes when switching between mobile and desktop
+      if (isNowMobile !== isMobile) {
+        document.querySelectorAll(".stat-card.active").forEach((card) => {
+          card.classList.remove("active");
+        });
+      }
+    });
+  });
+</script>
+
+<style>
+  .stats-grid {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 24px;
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 2rem;
+    font-family: "Poppins", sans-serif;
+  }
+
+  .stat-card {
+    background: #fff;
+    border: 1px solid #e2e8f0;
+    border-radius: 12px;
+    padding: 40px;
+    box-shadow: 0px 4px 6px -1px rgba(0, 0, 0, 0.1);
+    height: 200px; /* Fixed height for both states */
+    position: relative;
+    overflow: hidden;
+    font-family: "Poppins", sans-serif;
+  }
+
+  /* Desktop hover effects */
+  @media (min-width: 769px) {
+    .stat-card:hover .stat-info h3 {
+      transform: translateY(-10px);
+    }
+
+    .stat-card:hover .title-row {
+      transform: translateY(-10px);
+      margin-bottom: 0;
+    }
+
+    .stat-card:hover .stat-details {
+      opacity: 1;
+    }
+
+    .stat-card {
+      cursor: default; /* Remove pointer cursor on desktop */
+    }
+  }
+
+  .stat-content {
+    display: flex;
+    gap: 24px;
+    align-items: center; /* Center items vertically */
+    height: 100%;
+  }
+
+  .stat-icon {
+    flex-shrink: 0;
+    width: 64px; /* Bigger icon */
+    height: 64px; /* Bigger icon */
+  }
+
+  .stat-info {
+    flex-grow: 1;
+    width: calc(100% - 88px); /* Adjusted for larger icon */
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    justify-content: center; /* Center text vertically */
+    height: 100%;
+  }
+
+  .title-row {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    transition: transform 0.3s ease;
+    height: auto;
+    line-height: 1.2;
+    margin-bottom: 0;
+  }
+
+  .toggle-arrow {
+    display: none; /* Hide in web view */
+    color: #6b46c1;
+    font-size: 12px;
+    transition: transform 0.3s ease;
+  }
+
+  .stat-info h3 {
+    font-size: 24px;
+    font-weight: 600;
+    color: #1a202c;
+    margin: 0;
+    transition: transform 0.3s ease;
+    font-family: "Poppins", sans-serif;
+  }
+
+  .stat-details {
+    margin-top: 12px;
+    font-size: 16px;
+    color: #4a5568;
+    opacity: 0;
+    transition: opacity 0.3s ease;
+    position: absolute;
+    top: 40px; /* Position closer to title */
+    left: 0;
+    width: 100%;
+    font-family: "Poppins", sans-serif;
+  }
+
+  /* Mobile active class (for click) */
+  .stat-card.active .stat-info h3 {
+    transform: translateY(-10px); /* Reduced movement */
+  }
+
+  .stat-card.active .title-row {
+    transform: translateY(-10px); /* Reduced movement */
+    margin-bottom: 0;
+  }
+
+  .stat-card.active .stat-details {
+    opacity: 1;
+  }
```

Comment:
1) reduced movement? wdym? looks the same to me
2) you don't need to define these separately, can add them to the previous declarations:

```css
.stat-card:hover .stat-info h3,
.stat-card.active .stat-info h3 {
  /* .... stuff ... */
}
```